R/zchunk_LA101.ag_FAO_R_C_Y.R

```diff
@@ -40,79 +44,131 @@ module_aglu_LA101.ag_FAO_R_C_Y_DISABLED <- function(command, ...) {
     L100.FAO_ag_HA_ha <- get_data(all_data, "L100.FAO_ag_HA_ha")
     L100.FAO_ag_Prod_t <- get_data(all_data, "L100.FAO_ag_Prod_t")
 
-    # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
-    #
-    # NOTE: there are `merge` and/or 'match' calls in this code. Be careful!
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Merge-and-Match
-    # ===================================================
-
+    # Process FAO food consumption data (tons): remove unnecessary columns, convert units, aggregate to region and commodity
+    L100.FAO_ag_Food_t %>% select( -countries, -country.codes, -item.codes, -element, -element.codes ) %>%           # Remove unnecessary columns
+      left_join_error_no_match( iso_GCAM_regID, by=c( "iso") )  %>%                                                  # Map in ISO codes
```

Comment:
Do you mind doing a find-and-replace to remove spaces after open, and before close, parentheses? I.e. not do GCAM-style?